Dockerfile

```diff
@@ -1,72 +1,57 @@
-# ============== build stage ==================
-FROM debian as builder
+FROM debian:buster-20201117
 
-WORKDIR /root
+ADD https://dl.winehq.org/wine-builds/winehq.key \
+    https://download.opensuse.org/repositories/Emulators:/Wine:/Debian/Debian_10/Release.key \
+    https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks \
+    /tmp/
+
+COPY winetricks.sh entrypoint.sh /root/
```

Comment:
I'm not yet a docker pro as you might have already guessed :D but doesn't it make sense to move this line as late as possible ? As a rebuild with changes on those scripts might trigger other stages that might not be needed to rebuild.

Reply:
Yeah, agree. The problem is: is a draft! XD I didn't finished it yet and I am not even sure if it works (yet)

Reply:
i tested to build yesterday and it hung up somewhere on a wine step but i also had some connectivity problem here so maybe that was the reason, i'll try to build again later so i can check for any issues.

Reply:
Hangs in the last step: `/root/winetricks.sh --force -q dotnet48`
My knowledge of wine is 0, so I am stuck there. "Funny" fact is that running the command in a docker container myself it hangs also 🤔

Reply:
My last log lines:
```
Executing cd /root/.cache/winetricks/dotnet40
Executing wine dotNetFx40_Full_x86_x64.exe
```

i think you are missing the important downgrade steps

Reply:
do not execute the bash again, just winetricks --force -q dotnet48

Reply:
yep, that did it. Right now the version I have is 2Gb smaller than the master branch. I am doing another little trick to see if it can be improved further